models/skills.js

```diff
@@ -0,0 +1,72 @@
+const firestore = require("../utils/firestore");
+const skillsModel = firestore.collection("skills");
```

Comment:
This variable can be named better
```suggestion
const skillsCollection = firestore.collection("skills");
```